lib/matplotlib/backends/wx_compat.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+"""
+A wx API adapter to hide differences between wxPython classic and phoenix.
+
+It is assumed that the user code is selecting what version it wants to use,
+here we just ensure that it meets the minimum required by matplotlib.
+
+For an example see embedding_in_wx2.py
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import six
+
+missingwx = "Matplotlib backend_wx and backend_wxagg require wxPython >=2.8.12"
+
+
+try:
+    import wx
+    backend_version = wx.VERSION_STRING
+    is_phoenix = 'phoenix' in wx.PlatformInfo
+except ImportError:
+    raise ImportError(missingwx)
+
+# Ensure we have the correct version imported
+if wx.VERSION_STRING < "2.8.12":
```

Comment:
Don't do string comparisons for version checking. Use `distutils.version.LooseVersion`.